frontend/src/App.tsx

```
@@ -171,8 +172,20 @@ function AppContent({
 
       {/* Main content */}
       <main id="main-content" tabIndex={-1}>
-        <Suspense fallback={<div className="flex items-center justify-center min-h-screen">Loading...</div>}>
-          <Routes>
+        <ErrorBoundary>
+          <Suspense
+            fallback={
+              <div
+                className="flex items-center justify-center min-h-screen"
+                role="status"
+                aria-live="polite"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)